unity-client/Assets/Scripts/MainScripts/DCL/Models/Protocol/Engineinterface.cs

```diff
@@ -3423,16 +3423,16 @@ public sealed partial class PB_Vector3 : pb::IMessage<PB_Vector3> {
           default:
             _unknownFields = pb::UnknownFieldSet.MergeFieldFrom(_unknownFields, input);
             break;
-          case 13: {
-            X = input.ReadFloat();
+          case 9: {
```

Comment:
Where do these numbers come from? Is this code autogenerated?

Reply:
Yep, this should be autogenerated by the protobuf compiler